test/Microsoft.AspNetCore.SignalR.Tests/EndToEndTests.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Net.WebSockets;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Microsoft.AspNetCore.SignalR.Tests
+{
+    [CollectionDefinition(Name)]
+    public class EndToEndTestsCollection : ICollectionFixture<ServerFixture>
+    {
+        public const string Name = "EndToEndTests";
+    }
+
+    [Collection(EndToEndTestsCollection.Name)]
+    public class EndToEndTests
+    {
+        private readonly ServerFixture _serverFixture;
+
+        public EndToEndTests(ServerFixture serverFixture)
+        {
+            _serverFixture = serverFixture;
+        }
+
+        [Fact]
+        public async Task WebSocketsTest()
```

Comment:
Look at this file https://github.com/aspnet/Testing/blob/b912b8ed9cdacac0b1cfee013c7f1c1e69bf89cf/test/Microsoft.AspNetCore.Testing.Tests/OSSkipConditionTest.cs